### DISCUSSION TITLE
Help with specific installation error

### DISCUSSION THREAD

**Discussion by DuhBatista** [akvorado/akvorado#143] [Q&A]
Hello friends.

I'm having trouble with this impasse. Any tips to resolve. I'm trying to install "Compilation from source"
output for analysis.

root@debian:/home/nn/akvorado-1.5.7# make
▶ building golang.org/x/tools/cmd/goimports@latest…
go: downloading golang.org/x/tools v0.1.12
go: downloading golang.org/x/sys v0.0.0-20220722155257-8c9f86f7a55f
go: downloading golang.org/x/mod v0.6.0-dev.0.20220419223038-86c51ed26bb4
▶ formatting Go code…
▶ fetching node modules…
[vue-echarts] Switched to Vue 3 environment.
added 294 packages in 29,159s
▶ formatting JS code…
▶ building github.com/mgechev/revive@latest…
go: downloading github.com/mgechev/revive v1.2.3
go: downloading github.com/fatih/color v1.13.0
go: downloading github.com/mitchellh/go-homedir v1.1.0
go: downloading github.com/mattn/go-colorable v0.1.9
go: downloading github.com/mattn/go-isatty v0.0.14
go: downloading github.com/BurntSushi/toml v1.2.0
go: downloading github.com/mgechev/dots v0.0.0-20210922191527-e955255bf517
go: downloading github.com/pkg/errors v0.9.1
go: downloading github.com/chavacava/garif v0.0.0-20220630083739-93517212f375
go: downloading github.com/olekukonko/tablewriter v0.0.5
go: downloading github.com/fatih/structtag v1.2.0
go: downloading github.com/mattn/go-runewidth v0.0.9
▶ running golint…
▶ running jslint…
▶ building google.golang.org/protobuf/cmd/protoc-gen-go@v1.28.0…
go: downloading google.golang.org/protobuf v1.28.0
▶ compiling protocol buffers definition…
**Unknown flag: --go_opt
Makefile:72: recipe for target 'inlet/flow/decoder/flow-2.pb.go' failed
make: *** [inlet/flow/decoder/flow-2.pb.go] Error 1**

**Comment by vincentbernat** [comment]
What's your version of `protoc`? (`protoc --version`)

**Reply by vincentbernat** [reply]
It's likely too old. Alternatively, you can apply this patch: 0be3cbc3afad

**Reply by DuhBatista** [reply]
I redid everything again, I had an old protoc version!

new exit:
root@debian:/home/nn/akvorado-1.5.7# node --version
v16.17.0
root@debian:/home/nn/akvorado-1.5.7# npm --version
8.15.0
root@debian:/home/nn/akvorado-1.5.7# go version
go version go1.19 linux/amd64
root@debian:/home/nn/akvorado-1.5.7# protoc --version
libprotoc 3.21.6

The installation process by Compilation from source by documentation was now correct.

A silly question after the "make" command is finished without errors, what is the process to access the akvorado web interface? Do you need any more installation of pending issues?
just follow what you ask on this page. https://demo.akvorado.net/docs/install#pre-built-binary

**Reply by vincentbernat** [reply]
Any reason you do not want to use `docker-compose`? If you want to deploy yourself, look at `docker-compose.yml` for clues. You need to run the inlet, the orchestrator and the console components.

**Reply by DuhBatista** [reply]
Dear colleague,
upload the service via docker-compose.

Thank you for your wonderful tip.

Now I'm going to see how to do this collection with mikrotik.
